internal/provider/kubernetes/routes.go

```diff
@@ -274,6 +275,11 @@ func (r *gatewayAPIReconciler) processHTTPRoutes(ctx context.Context, gatewayNam
 							"name", refGrant.Name)
 					}
 				}
+
+				targetBackend := gatewayapi.GetTargetBackendReference(backendRef.BackendRef, gatewayapi.NamespaceDerefOr(gatewayapi.NamespacePtr(httpRoute.Namespace), "default"))
```

Comment:
can we run `process` in `controller.go` similar to the rest of the policies 
```
// Add the referenced Secrets and ConfigMaps in BackendTLSPolicies to the resourceTree
	r.processBackendTLSPolicySecretRefs(ctx, resourceTree, resourceMap)
```

Reply:
added processBackendTLSPolicyConfigMapRefs